nengo_spinnaker/utils/model.py

```diff
@@ -70,6 +71,38 @@ def find_connected_nodes(node, all_children=None):
     return force_removal
 
 
+def get_passthrough_node_dependencies(model, passthrough_nodes):
+    """Create a dictionary mapping from nodes and operators to all the nodes
+    and operators which transmit to it.
+
+    The resultant dictionary may be used to topologically sort the passthrough
+    Nodes.
+    """
+    # Create an empty dependency dictionary
+    deps = {n_op: set() for n_op in iteritems(passthrough_nodes)}
```

Comment:
I need to ensure that there is an entry for every Node I care about; so it's this or accessing every element of the `defaultdict` afterward.